django/applications/catmaid/admin.py

```diff
@@ -12,7 +13,7 @@
 from django.contrib.auth.admin import UserAdmin, GroupAdmin
 from django.contrib.auth.models import User, Group
 from django.utils.safestring import mark_safe
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext_lazy
```

Comment:
Hoping this is okay - having a function called _ is mysterious!

Reply:
I chatted about this with a local colleague - while this isn't common in Python, apparently in C, this is a tradition for naming internationalisation string-transform functions. It hasn't caught on much in Python because Python has a tradition of assigning unused returns from a function to a local variable _ as a shorthand for "ignore this".

Let me know if you want this part reverted back to _

Reply:
Explicit is better than implicit! Keep the full name, IMO.

Reply:
I agree. The reason why I did use the underscore alias is because the Django docs [suggest](https://docs.djangoproject.com/en/2.1/topics/i18n/translation/#standard-translation) it and it seems in fact quite common for translation functions in the general Python community. Given that we don't really do a lot of translation and using `ugettext_lazy` directly won't decrease readability here, I am on board with the explicit version.